internal/telegram/buttons.go

```
@@ -101,8 +101,10 @@
 	if user.Wallet != nil {
 		amount, err := bot.GetUserBalanceCached(user)
 		if err == nil {
-			log.Tracef("[appendMainMenu] user %s balance %d sat", GetUserStr(user.Telegram), amount)
-			MainMenuCommandBalance := fmt.Sprintf("%s %s", MainMenuCommandBalance, thirdparty.FormatSatsWithLKR(amount))
+
+			log.Tracef("[appendMainMenu] user %s balance %d sat(s)", GetUserStr(user.Telegram), amount)
+			MainMenuCommandBalance := fmt.Sprintf("%s %d sat(s)", MainMenuCommandBalance, thirdparty.FormatSatsWithLKR(amount))
```

golangci-lint (1.64.8)

printf: fmt.Sprintf format %d has arg thirdparty.FormatSatsWithLKR(amount) of wrong type string

(govet)